src/web-ui/buildspec.yml

```diff
@@ -2,25 +2,22 @@ version: 0.2
 phases:
   install:
     runtime-versions:
-      nodejs: 10
+      nodejs: 14
   pre_build:
     commands:
       - echo Installing NPM dependencies...
       - cd src/web-ui
-      - npm install
+      - npm ci
```

Comment:
`npm ci` is just more preferred on automated environments: https://docs.npmjs.com/cli/v7/commands/npm-ci